README.md

```diff
@@ -22,7 +22,10 @@ bash ~/Downloads/strap.sh
 rm -f ~/Downloads/strap.sh
 git clone https://github.com/crimethinc/website.git
 cd website
+./script/bootstrap
+source ~/.bash_profile
 ./script/setup
+overcommit --install
```

Comment:
I checked and `overcommit` seems to be a gem, so maybe we should add it to the `Gemfile`, tho?